shared/lib/themes/helpers.js

```diff
@@ -9,6 +9,13 @@ var analytics = require( 'analytics' ),
 var ThemesHelpers = {
 	oldShowcaseUrl: '//wordpress.com/themes/',
 
+	getSignupUrl( theme ) {
+		// TODO:
+		// * Possibly add a `source` parameter
```

Comment:
This was about adding `source` as a param to `getSignupUrl` (in case we want to invoke that function from other places than the showcase), but that might cause some trouble with `theme-options` and how we generate actions and URLs by always passing `theme` and `site` to `get{action}Url()`. I think I'll remove that comment, since YAGNI.